lib/ex_doc/formatter/html/templates/head_template.eex

```diff
@@ -2,18 +2,22 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <%= cond do %>
-    <%= page.id == :module -> %>
-    <title><%= page.content %></title>
-    <%= page.id == :readme -> %>
-    <title><%= page.content.project %> v<%= page.content.version %> README</title>
-    <%= page.id == :overview -> %>
-    <title><%= page.content.project %> v<%= page.content.version %> Overview</title>
+      <%= page.id == :module -> %>
+        <%= if page.module == config.project do %>
```

Comment:
Actually, it is fine to drop this. I would rather be consistent with "Module - Project Version" then special case when they are equal. Typically speaking, if we can avoid special cases, we should avoid them. :)